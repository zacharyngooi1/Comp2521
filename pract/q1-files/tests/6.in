3 7 10 13 24
2 5 8 70
