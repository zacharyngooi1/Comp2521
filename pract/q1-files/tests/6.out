list1: 3, 7, 10, 13, 24
list2: 2, 5, 8, 70
answer List: 2, 3, 5, 7, 8, 10, 13, 24, 70
